### PR TITLE
chore: add doc about upgrade bytebase in Kubernetes

### PR DESCRIPTION
**Pull request by dragonly**

### CODE REVIEW

File: content/docs/en/get-started/install/deploy-to-kubernetes.md [hunk 1]
```diff
@@ -120,6 +121,26 @@ For production setup, you should [configure a proper --external-url](/docs/get-s
...
 </hint-block>
 
+## Upgrade
+
+When a new Bytebase release is published, you can change the image version in the yaml file
+
+```yaml
+      containers:
+        - name: bytebase
+          image: bytebase/bytebase:<PUT THE NEXT VERSION HERE>
```
Comment (d-bytebase) [inline content/docs/en/get-started/install/deploy-to-kubernetes.md:131]: we can still use
          image: bytebase/bytebase:%%bb_version%%

as that is the version we want users to be on.

File: content/docs/en/get-started/install/deploy-to-kubernetes.md [hunk 2]
```diff
@@ -120,6 +121,26 @@ For production setup, you should [configure a proper --external-url](/docs/get-s
...
+```
+
+Sometimes we need to update the image to the latest digest, but without changing the image name and version. For example, you are using the `bytebase/bytebase:release-ci` image, whose version never changes.
```
Comment (d-bytebase) [inline content/docs/en/get-started/install/deploy-to-kubernetes.md:134]: This is not something we'd like to give to our customers typically.

File: content/docs/en/get-started/install/deploy-to-kubernetes.md (hunk 1 above)
Comment (tianzhou) [reply content/docs/en/get-started/install/deploy-to-kubernetes.md:131]: Here is the magic: https://sourcegraph.com/github.com/bytebase/bytebase.com/-/blob/nuxt.config.js?L420:42&subtree=true

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**